### ISSUE TITLE
Mod incompatible with Forestry

### ISSUE THREAD

**Issue by Cy4nX** [gigaherz/Survivalist#71]
Survivalist (1.12.2-2.1.4) does not remove plank recipes or add chopping block recipes for Forestry or Binnie's Mods.

**Comment by gigaherz** [comment]
In 1.12 and onward, recipes are loaded from JSON files. Forge gives a way to specify custom recipe factories, so that the JSON files can reference other recipe parsers besides the vanilla ones. After those JSON files are loaded, Forge notifies mods so that they can tweak/replace existing recipes. Mods are never supposed to do recipe registration in code, let alone after that event has fired.

I took a look at Forestry, and it registers its recipes manually, in code, and on top of that, in the "init" phase -- that is, after forge has loaded all the other recipes and sent the recipe registration events. This is bad practice and discouraged by forge, and it means it's running after all my code has already scanned for recipes.

I haven't checked Binnie's Mods yet, but I'm going to guess that they do the same.

So in short: It cannot be fixed without completely changing the way I handle recipes, which I'm not willing to do. Sorry.
